test/client.test.js

```diff
@@ -3767,7 +3803,7 @@ describe('ACC Client', function () {
             const schema = client.newSchema(xml);
             const jobs = schema.root.children["jobs"];
             expect(jobs.target).toBe("xtk:job");
-            await expect(jobs.linkTarget()).rejects.toThrow("Cannot read property 'getSchema' of null");
+            await expect(jobs.linkTarget()).rejects.toThrow("Cannot read properties of null (reading 'getSchema')");
```

Comment:
This is not a good error message. Why is there such a change in the error message: it does not seem related to your change at all and this is suspicious

Reply:
Sorry, that' still not good. First, it seems to work perfectly on 1.1.24, so this change is probably not necessary
If not can you explain why you changed the test vs. the code. AFAIK, it a test fails, it's s sign of a potential regression. Changing the test merely hides the issue and does not necessarily fixes it. Could you explain the reason of this failure and why you decided to change the test instead of the code?

Reply:
In any cases, it should be a separate PR as it is not related to the temp:group schema support

Reply:
Currently we observe this regression on acc web ui :
![image](https://user-images.githubusercontent.com/108726886/231391620-297ed4c7-0177-4ecc-92f7-ffae08fd56e9.png)

Reply:
I have reverted the change on the test. It is a regression that has present at least since version 1.1.23 : I tested on versions 1.1.24 and  1.1.25 the test fails.